src/renderer/components/mixins.scss

```diff
@@ -60,14 +60,19 @@
   }
 }
 
-// Makes custom font available at earlier stages (start preloading)
-// Element must be in DOM and contain some text of loading font.
-@mixin font-preload() {
+// Makes custom @font-family available at earlier stages.
+// Element must exist in DOM as soon as possible to initiate preloading.
+@mixin font-preload($fontFamily: "RobotoMono") {
```

Comment:
Do we need to specify the default param value here?

Reply:
Nope, provided just as an example.

Reply:
Would it be better to use doc comments?

 https://sass-lang.com/documentation/syntax/comments#documentation-comments

Reply:
Nope, it wouldn't better.. 
I can remove the default value but it really doesn't matter _(code should be self-documented as possible imho)._